src/sentry/web/frontend/demo_start.py

```diff
@@ -68,5 +69,15 @@ def dispatch(self, request):
         # delete all DSNs for the org so people don't send events
         ProjectKey.objects.filter(project__organization=org).delete()
 
+        # delete everything after 24 hours
+        countdown = 24 * 60 * 60
```

Comment:
It would be better to have this as a scheduled task that looks for orgs older than 24 hrs.

Using the `countdown` thing is not great since it means we have to keep the celery task scheduler file around

Reply:
@EvanPurkhiser Interesting idea, so maybe run a task periodically (once every hour) and look for old orgs?

Reply:
Yeah